components/scream/src/physics/rrtmgp/rrtmgp_inputs_initializer.hpp

```diff
@@ -0,0 +1,37 @@
+#ifndef SCREAM_RRTMGP_INPUTS_INITIALIZER_HPP
+#define SCREAM_RRTMGP_INPUTS_INITIALIZER_HPP
+
+#include "share/field/field_initializer.hpp"
+
+namespace scream {
+
+class RRTMGPInputsInitializer : public FieldInitializer
+{
+public:
+
+  virtual ~RRTMGPInputsInitializer () = default;
+
+  // The name of the initializer
+  std::string name () const { return "RRTMGPInputsInitializer"; }
+  
+  // Initialize fields
+  void initialize_fields ();
```

Comment:
You can annotate this (and other methods you've overridden from the `FieldInitializer` base class) with the `override` keyword to make it clearer that you're just implementing an interface here.

Reply:
Ahhh, that's slick, I love clarity. I think I implemented what you're talking about in the latest commit, let me know if it looks funny. Thanks!

Reply:
Hmm, I don't see it. Did you just add the `override` keyword at the end of the method declaration (before the `;` above)?

Reply:
https://github.com/E3SM-Project/scream/pull/810/commits/273f1d148b78878308fbab05256b7dd415998fb8

Reply:
I like using the override keyword. In 99% of the cases it doesn't catch bugs, but it definitely conveys the message.

Just one quick note: if I add it to one of the methods, then I have to add it to all the overridden methods, or else the compiler will issue a warning. Not a big deal, but worth remembering when trying to keep the compiler output under control.

Reply:
Awesome, thanks @bartgol ! I appreciate the learning opportunity!